### ISSUE TITLE
Compatible With Ios 10?

### ISSUE THREAD

**Issue by AppleTechy** [stefanesser/dumpdecrypted#20]
Is this compatible with ios 10? Don't want to accidentally brick my device or anything... Thanks :)

**Comment by theronyouknow** [comment]
Yes, this program is compatible with iOS 10. Make the dylib in accordance with the directions and move to the phone. Once the dylib is on the phone, sign it with this command: "ldid -S dumpdecrypted.dylib". Tested on a rooted 10.2 device. 

Note if you are going to try to decrypt applications with a lot of swift, such as twitter or facebook, they will have a large number of binary's in the Framework folder. You may want to try using this fork of dump decrypted by conradev: https://github.com/conradev/dumpdecrypted. Same build and use.

**Comment by xoceunder** [comment]
how can I modify an ios app and download it from the App Store
